docker-compose.yaml

```diff
@@ -95,4 +95,4 @@ networks:
     ipam:
       driver: default
       config:
-          - subnet: 10.200.0.0/24
\ No newline at end of file
+          - subnet: 10.200.0.0/24
```

Comment:
good catch 🎉

Reply:
Also, commenting here since I can't on the submodule itself: it looks like `src/mobilecoin` was also removed in this commit. Was that intentional?